docs/update_and_migration/from_4.3/update_from_4.3_old_commerce.md

```diff
@@ -0,0 +1,211 @@
+
+=== "[[= product_name_content =]]"
+
+    ``` json
+    "require":{
+        "ibexa/commerce-base-design": "4.4.0",
+        "ibexa/commerce-checkout": "4.4.0",
+        "ibexa/commerce-fieldtypes": "4.4.0",
+        "ibexa/commerce-price-engine": "4.4.0",
+        "ibexa/commerce-shop": "4.4.0",
+        "ibexa/commerce-shop-ui": "4.4.0",
+        "ezsystems/apache-tika-bundle": "^2.0",
+        "ezsystems/comment-bundle": "^3.1",
+        "ezsystems/job-queue-bundle": "^4.0",
+        "ezsystems/payment-core-bundle": "^3.0",
+        "ezsystems/stash-bundle": "^0.9",
+    }
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` json
+    "require":{
+        "ibexa/commerce-base-design": "4.4.0",
+        "ibexa/commerce-checkout": "4.4.0",
+        "ibexa/commerce-fieldtypes": "4.4.0",
+        "ibexa/commerce-price-engine": "4.4.0",
+        "ibexa/commerce-shop": "4.4.0",
+        "ibexa/commerce-shop-ui": "4.4.0",
+        "ezsystems/apache-tika-bundle": "^2.0",
+        "ezsystems/comment-bundle": "^3.1",
+        "ezsystems/job-queue-bundle": "^4.0",
+        "ezsystems/payment-core-bundle": "^3.0",
+        "ezsystems/stash-bundle": "^0.9",
+    }
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` json
+    "require":{
+        "ibexa/commerce-base-design": "4.4.0",
+        "ibexa/commerce-checkout": "4.4.0",
+        "ibexa/commerce-fieldtypes": "4.4.0",
+        "ibexa/commerce-price-engine": "4.4.0",
+        "ibexa/commerce-shop": "4.4.0",
+        "ibexa/commerce-shop-ui": "4.4.0",
+        "ezsystems/apache-tika-bundle": "^2.0",
+        "ezsystems/comment-bundle": "^3.1",
+        "ezsystems/job-queue-bundle": "^4.0",
+        "ezsystems/payment-core-bundle": "^3.0",
+        "ezsystems/stash-bundle": "^0.9",
+        "ibexa/commerce-admin-ui": "4.4.0",
+        "ibexa/commerce-erp-admin": "4.4.0",
+        "ibexa/commerce-order-history": "4.4.0",
+        "ibexa/commerce-page-builder": "4.4.0",
+        "ibexa/commerce-rest": "4.4.0",
+        "ibexa/commerce-transaction": "4.4.0"
+    }
+    ```
+
+Next, remove the entries with new packages alongside with routing and configuration in `config/routes/ibexa_cart.yaml`, `config/routes/ibexa_checkout.yaml`and `config/routes/ibexa_storefront.yaml`:
+
+```php
+    Ibexa\Bundle\Cart\IbexaCartBundle::class => ['all' => true],
+    Ibexa\Bundle\Checkout\IbexaCheckoutBundle::class => ['all' => true],
+    Ibexa\Bundle\Storefront\IbexaStorefrontBundle::class => ['all' => true],
+```
+
+### Update the database
+
+Next, update the database if you are using Ibexa Commerce.
+Ibexa Content and Ibexa Experience do not require the database update.
+
+[[% include 'snippets/update/db/db_backup_warning.md' %]]
+
+Apply the following database update scripts:
+
+=== "MySQL"
+
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/commerce/ibexa-4.3.latest-to-4.4.0.sql
```

Comment:
Since the upgrade script only has Cart tables at the moment it will add them even though clients follwing this guide won't be using it. Not ideal but I think at least we can assured they have exactly the schema we want and all future upgrade SQLs will work.

cc @adamwojs